### ISSUE TITLE
Unable to Start Bitgo Express

### ISSUE THREAD

**Issue by HolluwaTosin360** [BitGo/BitGoJS#289]
I received this error:

Error: Cannot find module '../dist/src/expressApp'
    at Function.Module._resolveFilename (module.js:476:15)
    at Function.Module._load (module.js:424:25)
    at Module.require (module.js:504:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/bitpeer/BitGoJS/bin/bitgo-express:5:66)
    at Module._compile (module.js:577:32)
    at Object.Module._extensions..js (module.js:586:10)
    at Module.load (module.js:494:32)
    at tryModuleLoad (module.js:453:12)
    at Function.Module._load (module.js:445:3)
    at Module.runMain (module.js:611:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:160:9)
    at bootstrap_node.js:507:3

**Comment by tylerlevine** [comment]
Can you reproduce this issue using the latest master, commit `2108a2e3b945f42594288e5a87b06be80b5e64d0`?

**Comment by HolluwaTosin360** [comment]
These are the commands i ran:

》git clone https://www.github.com/BitGo/BitGoJS

》cd BitGoJS

》npm install

》cd bin

》./bitigo-express

**Comment by tylerlevine** [comment]
Are you installing as root?

**Comment by HolluwaTosin360** [comment]
Yes, i have been running installations for a while now.

**Comment by tylerlevine** [comment]
BitGoJS should not be installed using `root` (use `npm install --unsafe-perm` if you must install as root for now), but the issue you are experiencing is a regression and will be fixed in our next npm release, 5.0.0

**Comment by HolluwaTosin360** [comment]
By installing as root i believe you meant install with super user access. If there is anyway i can get around this please let me know. Can i clone an older version instead, because it was working well just before recently.

**Comment by tylerlevine** [comment]
Yes, you can clone an older version using the following commands:
```
git clone https://github.com/BitGo/BitGoJS
cd BitGoJS
git checkout 4.49.1
npm install
cd bin
./bitgo-express
```

This will also be fixed in BitGoJS 5.0.0, which will be released to npm next week.
